helm/tenant/values.yaml

```diff
@@ -28,14 +28,12 @@ tenant:
     pullPolicy: IfNotPresent
   ## Customize any private registry image pull secret.
   ## currently only one secret registry is supported
-  imagePullSecret: { }
+  imagePullSecret: {}
   ## If a scheduler is specified here, Tenant pods will be dispatched by specified scheduler.
   ## If not specified, the Tenant pods will be dispatched by default scheduler.
-  scheduler: { }
+  scheduler: {}
   ## Secret name that contains additional environment variable configurations.
   ## The secret is expected to have a key named config.env containing environment variables exports.
-  configuration:
-    name: myminio-env-configuration
```

Comment:
don't delete this.

Reply:
That part will become useless due to this code, there is no reference anymore to configuration.name in tenant.yaml:
```yaml
  ## Secret with default environment variable configurations
  configuration:
    name: {{ .secrets.existingSecret | default .secrets.name }}
```